test/parallel/test-sqlite-backup.mjs

```diff
@@ -0,0 +1,229 @@
+import * as common from '../common/index.mjs';
+import tmpdir from '../common/tmpdir.js';
+import { join } from 'path';
+import { DatabaseSync } from 'node:sqlite';
+import { describe, test } from 'node:test';
+import { writeFileSync } from 'fs';
+
+let cnt = 0;
+
+tmpdir.refresh();
+
+function nextDb() {
+  return join(tmpdir.path, `database-${cnt++}.db`);
+}
+
+function makeSourceDb() {
+  const database = new DatabaseSync(':memory:');
+
+  database.exec(`
+    CREATE TABLE data(
+      key INTEGER PRIMARY KEY,
+      value TEXT
+    ) STRICT
+  `);
+
+  const insert = database.prepare('INSERT INTO data (key, value) VALUES (?, ?)');
+
+  for (let i = 1; i <= 2; i++) {
+    insert.run(i, `value-${i}`);
+  }
+
+  return database;
+}
+
+describe('DatabaseSync.prototype.backup()', () => {
+  test('throws if path is not a string', async (t) => {
+    const database = makeSourceDb();
+
+    t.assert.rejects(async () => {
+      await database.backup();
```

Comment:
Shouldn't these input validation tests throw instead of rejecting? That's how most Node APIs work on bad input.